controllers/app.controller.js

```diff
@@ -69,3 +70,17 @@ exports.patchArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  if (isNaN(comment_id)) {
+    return next({ status: 400, msg: 'Invalid comment_id' });
+  }
+  removeComment(comment_id)
+    .then((comment) => {
+      if (!comment.length) {
+        return Promise.reject({ status: 404, msg: 'Comment not found' });
+      }
+      res.status(204).send();
+    })
```

Comment:
This is also the same as the above comment, but this example also goes against how a .then(...) should work, as we should only enter the .then(...) when we are following the happy path and should not need to do conditional logic like this to throw an error